community-fund/COMMUNITY_FUND_POLICY.md

```diff
@@ -0,0 +1,79 @@
+# Cross Project Council Community Fund
+
+## About The Community Fund
+
+The OpenJS Foundation’s community fund supports travel and attendance at events and activities directly related to OpenJS projects, technologies, or goals. We support travel for individuals who are:
+
+* Collaborators
+* Standards body participants
+* Speakers
+
+The fund can cover the costs, in whole or in part, of a participant's hotel, transportation, and some event registrations.
+Applicants should explore employer-sponsored funding options as their first source of financial support.
+Additionally, 25% of the community fund is allocated specifically to support diverse participants.
+Funding decisions are determined by a combination of need and potential impact.
+
+## Community Fund Rules
+
+Valid travel expenses are:
+
+* Coach airfare tickets
+* Accommodations (Dates of the event plus one night prior are eligible for reimbursement)
+* Ground transportation to/from the airport
+
+NOTE: Funds may not be used for miscellaneous travel expenses including food, visa costs, non-airport transportation, baggage fees, etc.
```

Comment:
This appears to be focused entirely on transportation by air carriers. Let's at the very least explicitly cover climate-friendly alternatives, such as trains, as well.